src/components/ScholarshipDetailCard.jsx

```diff
@@ -111,7 +112,16 @@ export default function ScholarshipDetailCard({ scholarship }) {
         <Button
           variant="outlined"
           className={classes.shareBtn}
-          startIcon={<Share />}>
+          startIcon={<Share />}
+          onClick={() =>
+            history.replace({
+              state: {
+                showShareDialog: true,
+                shareTitle: `${amount?.toString()} - ${name}`,
+                shareURL: `https://${window.location.hostname}/scholarships/${scholarship.id}`,
```

Comment:
I tried doing something like
`<Button
component={Link} 
to={{state: showLoginDialog : true}}
replace/>` but it was not working. Not sure why.